Backpack/src/main/java/net/skyscanner/backpack/bottomsheet/BpkBottomSheetBehaviour.kt

```diff
@@ -61,6 +61,23 @@ class BpkBottomSheetBehaviour<V : View>(context: Context, attrs: AttributeSet? =
     )
   }
 
+  override fun onLayoutChild(parent: CoordinatorLayout, child: V, layoutDirection: Int): Boolean {
+    return super.onLayoutChild(parent, child, layoutDirection).apply { child.background = background }
+  }
+
+  override fun setState(state: Int) {
```

Comment:
similarly when the state was set before layout the background wasn't set correctly - this fixes that issue